buildkite/src/Jobs/Lint/Changelog.dhall

```diff
@@ -0,0 +1,49 @@
+let S = ../../Lib/SelectFiles.dhall
+
+let Cmd = ../../Lib/Cmds.dhall
+
+let Pipeline = ../../Pipeline/Dsl.dhall
+
+let PipelineTag = ../../Pipeline/Tag.dhall
+
+let JobSpec = ../../Pipeline/JobSpec.dhall
+
+let Command = ../../Command/Base.dhall
+
+let Size = ../../Command/Size.dhall
+
+let trigger = S.compile [ S.strictlyStart (S.contains "src") ]
+
+let reqFile = "^changes/\\\${BUILDKITE_PULL_REQUEST}-.*.md"
+
+in  Pipeline.build
+      Pipeline.Config::{
+      , spec = JobSpec::{
+        , dirtyWhen =
+          [ S.strictlyStart (S.contains "src")
+          , S.strictlyStart (S.contains "scripts/changelog.sh")
+          , S.strictlyStart (S.contains "buildkite/scripts/changelog.sh")
+          , S.strictlyStart (S.contains "scripts/github/github_info")
+          , S.exactly "buildkite/src/Jobs/Lint/Changelog" "dhall"
+          ]
+        , path = "Lint"
+        , name = "Changelog"
+        , tags =
+          [ PipelineTag.Type.Fast
+          , PipelineTag.Type.Lint
+          , PipelineTag.Type.Stable
+          ]
+        }
+      , steps =
+        [ Command.build
+            Command.Config::{
+            , commands =
+              [ Cmd.run
+                  "./buildkite/scripts/changelog.sh --trigger ${trigger} --required-change ${reqFile}"
```

Comment:
What will `trigger` be set to? Expression above looks quite complex, although I'd expect it to be just `src/` or something along these lines